deck/views.py

```diff
@@ -17,8 +17,14 @@ def _get_request_var(request, key, default=1):
         return request.GET.get(key, default)
 
 def get_jokers_enabled(request):
-    j = _get_request_var(request, 'jokers_enabled', 'false')
-    return j=='true'
+    j = _get_request_var(request, 'jokers_enabled')
+    if j != 'true':
+        if j == 'false':
+            return False
+        else:
+            return
+    else:
+        return True
```

Comment:
I don't understand why this needed to be updated.

Reply:
If we default to false (or true), then we wrongly assume that we should/shouldn't include jokers when shuffling. Whereas, if this function returns nothing, we know that the request to create or shuffle the deck was ambivalent about jokers. In that situation, we'd check `deck.include_jokers` and use the value there (which defaults to false).

Hopefully that makes sense?

Reply:
That makes sense. Can we make it more readable by doing:
```
j = _get_request_var(request, 'jokers_enabled')
if j == "true".lower(): return True
if j == "false".lower(): return False
return
```

Reply:
Yeah, absolutely. I get that updated.

Reply:
I moved the `.lower()` to `j` since "true" and "false" are already lowercase strings. I also wrapped the two ifs in another if that checks to make sure `j` is a string. When no query param is given, `_get_request_var` returns 1, which doesn't have a .lower property.

Reply:
Perfect! Thanks for thinking through all this rather than just blindly copy and pasting my awful code. I'll get this merged in. Thanks again for your work!